extensions/adobe/experience/customerJourneyManagement/message-experienceevent.example.5.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
should we name the examples to indicate;
EVENT_TYPE & CHANNEL & STATUS

this one looks like tracking-email-click example for instance

Reply:
The current automation scripts which validates the examples with the corresponding schema file requires, example files to have a naming convention : schema-name.example.[0-9].json